### DISCUSSION TITLE
YouTube replaced “Library” with “You” tab, wanna back to library

### DISCUSSION THREAD

**Discussion by dora727** [PoomSmart/YTABConfig#82] [Q&A]
1st image: before change
2nd image: after change
it’s depend on account tho, my alt already had it but not my main account (however i use alt to watch some contents and i don’t like this change)
![IMG_1601](https://github.com/PoomSmart/YTABConfig/assets/109880918/75bd8b89-4f8d-4817-82d9-1e22b41eca1a)
![IMG_1600](https://github.com/PoomSmart/YTABConfig/assets/109880918/71b027e1-d808-45ad-a0f5-bcc5dd31f236)

**Comment by dora727** [comment]
the new you tab if you guys wanna see
![IMG_1615](https://github.com/PoomSmart/YTABConfig/assets/109880918/dd50566d-daf1-4069-878d-efea0ba4f848)

**Reply by AlteredCabr0n** [reply]
Did you figure this out?

**Comment by EscapeAltDelete** [comment]
T̶u̶r̶n̶ ̶o̶f̶f̶ ̶t̶h̶e̶ ̶A̶/̶B̶ ̶f̶l̶a̶g̶:̶ ̶c̶h̶a̶n̶n̶e̶l̶s̶C̶l̶i̶e̶n̶t̶C̶o̶n̶f̶i̶g̶I̶o̶s̶C̶h̶a̶n̶n̶e̶l̶N̶a̶v̶R̶e̶s̶t̶r̶u̶c̶t̶u̶r̶i̶n̶g̶

Edit: Spoof YouTube to v18.34.5 and restart yt a couple times.

**Reply by migmit** [reply]
Doesn't work.

**Reply by EscapeAltDelete** [reply]
See original comment

T̶h̶a̶t̶'̶s̶ ̶r̶e̶a̶l̶l̶y̶ ̶w̶e̶i̶r̶d̶.

I̶ ̶t̶h̶i̶n̶k̶ ̶i̶ ̶m̶i̶g̶h̶t̶ ̶h̶a̶v̶e̶ ̶m̶i̶s̶t̶a̶k̶e̶n̶ ̶i̶t̶ ̶f̶o̶r̶ ̶w̶o̶r̶k̶i̶n̶g̶,̶ ̶n̶o̶w̶ ̶t̶h̶a̶t̶ ̶i̶ ̶t̶r̶y̶ ̶t̶o̶ ̶t̶u̶r̶n̶ ̶t̶h̶e̶ ̶f̶l̶a̶g̶ ̶b̶a̶c̶k̶ ̶o̶n̶ ̶t̶h̶e̶ ̶n̶e̶w̶ ̶u̶i̶ ̶d̶o̶e̶s̶n̶'̶t̶ ̶c̶o̶m̶e̶ ̶b̶a̶c̶k̶.

I̶'̶m̶ ̶n̶o̶t̶ ̶s̶u̶r̶e̶ ̶h̶o̶w̶ ̶i̶ ̶m̶a̶n̶a̶g̶e̶d̶ ̶t̶o̶ ̶c̶h̶a̶n̶g̶e̶ ̶m̶y̶ ̶l̶i̶b̶r̶a̶r̶y̶ ̶t̶o̶ ̶t̶h̶e̶ ̶o̶l̶d̶ ̶u̶i̶ ̶t̶h̶e̶n̶,̶ ̶a̶s̶ ̶i̶ ̶h̶a̶v̶e̶n̶'̶t̶ ̶c̶h̶a̶n̶g̶e̶d̶ ̶a̶n̶y̶ ̶o̶t̶h̶e̶r̶ ̶A̶/̶B̶ ̶f̶l̶a̶g̶s̶.

E̶d̶i̶t̶:̶ ̶I̶ ̶s̶p̶o̶o̶f̶e̶d̶ ̶t̶h̶e̶ ̶y̶o̶u̶t̶u̶b̶e̶ ̶v̶e̶r̶s̶i̶o̶n̶ ̶t̶o̶ ̶v̶1̶8̶.̶3̶4̶.̶5̶ ̶w̶h̶e̶n̶ ̶t̶r̶y̶i̶n̶g̶ ̶t̶o̶ ̶g̶e̶t̶ ̶t̶h̶e̶ ̶o̶l̶d̶ ̶l̶i̶b̶r̶a̶r̶y̶,̶ ̶w̶h̶e̶n̶ ̶t̶h̶a̶t̶ ̶d̶i̶d̶n̶'̶t̶ ̶w̶o̶r̶k̶ ̶i̶ ̶t̶r̶i̶e̶d̶ ̶c̶h̶a̶n̶g̶i̶n̶g̶ ̶t̶h̶e̶ ̶f̶l̶a̶g
w̶h̶i̶c̶h̶ ̶d̶i̶d̶ ̶w̶o̶r̶k̶.̶ ̶I̶ ̶t̶h̶i̶n̶k̶ ̶i̶t̶ ̶m̶i̶g̶h̶t̶ ̶h̶a̶v̶e̶ ̶s̶o̶m̶e̶t̶h̶i̶n̶g̶ ̶t̶o̶ ̶d̶o̶ ̶w̶i̶t̶h̶ ̶t̶u̶r̶n̶i̶n̶g̶ ̶t̶h̶e̶ ̶f̶l̶a̶g̶ ̶o̶f̶f̶ ̶w̶h̶i̶l̶e̶ ̶o̶n̶ ̶t̶h̶e̶ ̶s̶p̶o̶o̶f̶e̶d̶ ̶v̶e̶r̶s̶i̶o̶n̶.

N̶o̶t̶ ̶s̶u̶r̶e̶ ̶t̶h̶o̶u̶g̶h̶,̶ ̶y̶o̶u̶'̶l̶l̶ ̶h̶a̶v̶e̶ ̶t̶o̶ ̶t̶r̶y̶ ̶s̶o̶m̶e̶ ̶s̶t̶u̶f̶f̶ ̶o̶u̶t̶ ̶y̶o̶u̶r̶s̶e̶l̶f̶ ̶a̶s̶ ̶i̶t̶s̶ ̶a̶ ̶b̶i̶t̶ ̶l̶a̶t̶e̶ ̶f̶o̶r̶ ̶m̶e̶ ̶t̶o̶ ̶b̶e̶ ̶d̶o̶i̶n̶g̶ ̶t̶h̶i̶s̶.

̶E̶d̶i̶t̶ ̶2̶:̶ ̶I̶ ̶f̶u̶c̶k̶e̶d̶ ̶a̶r̶o̶u̶n̶d̶ ̶a̶ ̶b̶i̶t̶ ̶o̶n̶ ̶m̶y̶ ̶s̶e̶c̶o̶n̶d̶ ̶d̶e̶v̶i̶c̶e̶ ̶w̶i̶t̶h̶ ̶a̶ ̶f̶r̶e̶s̶h̶ ̶i̶p̶a̶,̶ ̶a̶n̶d̶ ̶i̶ ̶h̶a̶v̶e̶ ̶n̶o̶ ̶c̶l̶u̶e̶ ̶w̶h̶y̶ ̶i̶t̶ ̶w̶o̶r̶k̶s̶ ̶b̶u̶t̶ ̶t̶r̶y̶ ̶d̶o̶i̶n̶g̶ ̶e̶x̶a̶c̶t̶l̶y̶ ̶a̶s̶ ̶i̶ ̶d̶i̶d̶ ̶i̶n̶ ̶t̶h̶i̶s̶ ̶v̶i̶d̶e̶o̶:̶ ̶h̶t̶t̶p̶s̶:̶/̶/̶i̶.̶s̶u̶p̶a̶.̶c̶o̶d̶e̶s̶/̶H̶i̶r̶y̶X̶

**Comment by Gamerdragon0095** [comment]
In the past couple weeks or so (as of the time I’m writing this) the old pre v19 versions of YouTube have started to not let people log into their accounts on the app, so for those of us who want the history tab back, we’re really gonna have to figure this out, and the sooner the better IMO.

I did a little digging just to see if anything immediately looked promising just based off the name alone, and I’ve come up with a short list of potential fixes, but unfortunately I don’t have a lot of time to actually go through and test them out today, but I’ll be sure to come back here to share the results of my testing after I’m done with it!

**Reply by foxnoodles** [reply]
It has been a year, any newss?
